### ISSUE TITLE
Not resolving tsconfig `paths`

### ISSUE THREAD

**Issue by SimonCockx** [import-js/eslint-import-resolver-typescript#205]
I have two projects, `project-a` and `project-b`, one importing the other via a `paths` synonym defined in `tsconfig.base.json`. Although the TypeScript compiler knows how to resolve the import, this plugin is complaining that it cannot resolve the path.

I've read the documentation of both this plugin and the main import plugin, and I think I configured everything right, but with no luck. Is there something wrong with my setup?

Minimal repro:
- clone https://github.com/SimonCockx/eslint-import-bug
- run `npx eslint project-a/src/index.ts`

Snippets:

`tsconfig.base.ts`
```
"compilerOptions": {
  "paths": {
    "@importbug/b": ["project-b/src/index.ts"]
  },
  ...
}
```

`project-a/.eslintrc.json`
```
{
  "plugins": [
    "@typescript-eslint",
    "import"
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": ["project-a/tsconfig.json"],
    "sourceType": "module"
  },
  "settings": {
    "import/resolver": {
      "typescript": {
        "alwaysTryTypes": true,
        "project": "tsconfig.json",
        "extensions": [".ts"]
      }
    },
    "import/parsers": {
      "@typescript-eslint/parser": [
        ".ts", ".tsx"
      ]
    }
  },
  "extends": [
    "plugin:import/typescript", "plugin:import/recommended"
  ]
}
```

`project-a/src/index.ts`
```
import { b } from "@importbug/b";
                   ^^^^^^^^^^^
b;
```

**Comment by dgutoryan** [comment]
I have a similar issue but for build. IDE says that everything is ok. But when I run react project - `Can't resolve '@shared/context'`

**Comment by Nesvet** [comment]
Any updates?

**Comment by SimonCockx** [comment]
See a bit more info on the `eslint-plugin-import` repo: https://github.com/import-js/eslint-plugin-import/issues/2652#issuecomment-1666260315

No solution yet though...

**Comment by JounQin** [comment]
1. Use `https://github.com/un-es/eslint-plugin-i`
2. Use `yarn set resolution tsconfig-paths ^4` like https://github.com/import-js/eslint-plugin-import/issues/2652#issuecomment-1804087161

**Comment by SimonCockx** [comment]
Following the same reasoning as in https://github.com/import-js/eslint-plugin-import/issues/2652, sgould this be reopened? @ljharb?

**Comment by JounQin** [comment]
No, it has nothing related to this resolver.

**Comment by SimonCockx** [comment]
Alright, makes sense then.

**Comment by csantos1113** [comment]
> 1. Use `https://github.com/un-es/eslint-plugin-i`

I've tried using this but I can't get tsconfig.`paths` to work either 😞

**Comment by JounQin** [comment]
> > 1. Use `https://github.com/un-es/eslint-plugin-i`
> 
> I've tried using this but I can't get tsconfig.`paths` to work either 😞

Please raise a new issue with runnable reproduction instead.
